src/modules/profile/pages/profilePage/views/account/views/AccountInfo.tsx

```diff
@@ -41,14 +44,21 @@ export const AccountInfo = () => {
 
   const isViewingOwnProfile = useViewingOwnProfileAtomValue()
 
+  const { onCopy } = useClipboard(userProfile.username)
+  const toast = useNotification()
+
+  // Define color mode values
+  const heroIdBg = useColorModeValue('gray.100', 'gray.700')
+  const heroIdColor = useColorModeValue('neutral.500', 'neutral.600')
+  const heroIdHoverBg = useColorModeValue('gray.200', 'gray.600')
```

Comment:
Please use the palette from our design system.
We've stopped using neutral for neutral1 . ( This needs to be changed so that neutral1 is neutral once all pages are updated )

The gray is not in our design system, let try to avoid it unless really necessary.

You can check the pallette.js for our colors.

Reply:
Gray is in the colour palette though, so what does it get replaced with?